src/index.js

```diff
@@ -1,27 +1,37 @@
-import _ from 'lodash';
+import _, { reduce } from 'lodash';
 import './style.css';
 
-const items = [
-  {
-    description: 'Read books',
-    completed: '0',
-    index: 1,
-  },
-  {
-    description: 'Wash dishes',
-    completed: '1',
-    index: 2,
-  },
-  {
-    description: 'Cook food',
-    completed: '1',
-    index: 3,
-  },
-];
+let doLists = [];
 
+const newItem = document.querySelector('.newItem');
 const lists = document.querySelector('.lists');
 
-items.forEach((key) => {
+// check local storage is avilable
+function checklocal() {
+  if (localStorage.getItem('Do-Lists')) {
+    doLists = JSON.parse(localStorage.getItem('Do-Lists'));
+  } else {
+    localStorage.setItem('Do-Lists', JSON.stringify(doLists));
+  }
+}
+
+checklocal();
+
+// add new tasks
+newItem.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    const num = doLists.length;
+    const item = document.querySelector('.newItem').value;
+    doLists.push({ description: item, completed: false, index: num });
+    localStorage.setItem('Do-Lists', JSON.stringify(doLists));
+    window.location.reload();
+  }
+});
```

Comment:
- [ ] [OPTIONAL] - It would be a better user experience if when you submit a task, the input box is cleared and focused. This will greatly improve your UX of your project. Below is a sample of how this can be achieved.

```
newItem.value = ''
```